### PR TITLE
Hide vote details from non-admins.

### PR DESCRIPTION
**Pull request by zorbathut**
It is intentional that the links show up on admin level 2 but aren't usable until admin level 3; I don't know what I want right now and I want to encourage admins to pester me about it.

### CODE REVIEW

**Comment by zorbathut** [comment]
Fixes https://github.com/themotte/rDrama/issues/112 .

File: files/templates/userpage.html [hunk 1]
```diff
@@ -104,7 +104,9 @@ <h1 class="font-weight-bolder h3 mb-0"><span {% if u.patron %}class="patron" sty
 				{% else %}<pre></pre>
 				{% endif %}
 				
-				<div class="font-weight-bolder mb-2"><a class="mr-1" href="/@{{u.username}}/upvoters">Upvoters</a> | <a class="mx-1" href="/@{{u.username}}/downvoters">Downvoters</a> | <a class="mx-1" href="/@{{u.username}}/upvoting">Upvoted</a> | <a class="ml-1" href="/@{{u.username}}/downvoting">Downvoted</a></div>
+				{% if u.admin_level >= 2 %}
```
Comment (JulianRota) [inline files/templates/userpage.html]: Here and in the other change below in this file, you're using `u`, which is the user who's page you are currently viewing. Pretty sure this ought to be `v`, the user you are currently logged in as - currently anyone can see the links only if the user they're viewing is admin.

Comment (zorbathut) [reply files/templates/userpage.html]: Dangit, you're correct. Fixed.

**Review by JulianRota — APPROVED** [review]
Looks good now

**Pull request merged**